src/components/TabList.js

```diff
@@ -2,12 +2,21 @@ import React, { PropTypes } from 'react';
 import cx from 'classnames';
 
 function renderChildren(props) {
-  return React.Children.map(props.children, (child) =>
-    React.cloneElement(child, {
+  return React.Children.map(props.children, (child) => {
+    // if child is not a tab we don't need to clone it
+    // since we don't need to add custom props
+
+    if (child.type.displayName !== 'Tab') {
```

Comment:
Was there a reason to use the displayName. If not I think it would be better to do `child.type !== Type` and `import Tab from './Tab'` at the top.